Modules/vc-module-content/VirtoCommerce.ContentModule.Core/Model/FrontMatterHeaders.cs

```diff
@@ -2,13 +2,17 @@
 using VirtoCommerce.Platform.Core.Common;
 using VirtoCommerce.Platform.Core.DynamicProperties;
 
-namespace VirtoCommerce.ContentModule.Web
+namespace VirtoCommerce.ContentModule.Core.Model
```

Comment:
then need to delete VirtoCommerce.ContentModule.Web.Infrastructure.DynamicProperties